e2e/tests-dfx/delete.bash

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env bats
+
+load ../utils/_
+
+setup() {
+    standard_setup
+
+    dfx_new
+}
+
+teardown() {
+    dfx_stop
+    
+    standard_teardown
+}
+
+@test "delete can be used to delete a canister" {
+    dfx_start
+    dfx deploy e2e_project_backend
+    dfx canister stop e2e_project_backend
+    assert_command dfx canister delete e2e_project_backend
+    assert_command_fail dfx canister info e2e_project_backend
+    assert_contains "Cannot find canister id. Please issue 'dfx canister create e2e_project_backend'."
```

Comment:
This only tests if dfx locally saved the ID. Can you please also test if the canister does not exist anymore on the replica?